File: Implemented Funs Using C++/String Repeat.cpp
#include<iostream>
#include<string>
#include<algorithm>

using namespace std;

// Implemented Function Will Repeat The String
// value : The string that will be repeated
// repeatNum :The number of repeating

string stringRepeat(string value ,int repeatNum){
    string result;
    while(repeatNum--){
        if(repeatNum==0) result+=value;
        else result+=value+" ";
    }
    return result;
}

int main() {

  cout<<stringRepeat("Migo",5)<<endl;

    return 0;
}

File: Implemented Funs Using JS/String Repeat.js
// Implemented Function Will Repeat The String
// s : The string that will be repeated
// num :The number of repeating

//
function print() {
  console.log("Hello");
}
print();
function stringRepeat(s, num) {
  let result = "";
  while (num--) {
    result += s + " ";
  }
  return result;
}

console.log(stringRepeat("Migo", 5));
